Userland/Libraries/LibWeb/Loader/Resource.cpp

```diff
@@ -90,9 +90,6 @@ void Resource::did_load(Badge<ResourceLoader>, ReadonlyBytes data, const HashMap
         if (encoding.has_value()) {
             dbgln_if(RESOURCE_DEBUG, "Set encoding '{}' from Content-Type", encoding.has_value());
             m_encoding = encoding.value();
-        } else {
-            // FIXME: This doesn't seem nice.
-            m_encoding = "utf-8";
         }
```

Comment:
Let's unset the encoding explicitly if we can't find one from the content type and encoding (probably by adding a `m_encoding = {};` before this whole if)